raiden-cli/src/controllers/channels.ts

```diff
@@ -0,0 +1,81 @@
+import { Request, Response, NextFunction } from 'express';
+import { ChannelState } from 'raiden-ts';
+import _ from 'lodash';
```

Comment:
In the SDK/dApp we always import the functions directly e.g. for `map` that would be `import map from 'lodash/map';`. This is to enable tree-shaking anything unused when bundling instead of pulling the whole `lodash` in the final bundle

Reply:
That's a good point. Sometimes (especially for such generic names) I like to have a reference while reading the code where this comes from. But you are right, having the whole lodash in the package would be nasty. I'll fix that. :+1:

Reply:
:heavy_check_mark: